components/admin.tsx

```diff
@@ -0,0 +1,1103 @@
+			setReloadDatebase(false);
+		}
+	}, [reloadDatabase]);
+
+	const [passportSorting, setPassportSorting] = React.useState<SortingState>(
+		[],
+	);
+	const [passportColumnFilters, setPassportColumnFilters] =
+		React.useState<ColumnFiltersState>([]);
+	const [passportColumnVisibility, setPassportColumnVisibility] =
+		React.useState<VisibilityState>({});
+	const [passportRowSelection, setPassportRowSelection] = React.useState({});
+	const [passportPagination, setPassportPagination] = React.useState<PaginationState>({
+		pageIndex: 0,
+		pageSize: 5,
+	})
+
+	const [ceremonySorting, setCeremonySorting] = React.useState<SortingState>(
+		[],
+	);
+	const [ceremonyColumnFilters, setCeremonyColumnFilters] =
+		React.useState<ColumnFiltersState>([]);
+	const [ceremonyColumnVisibility, setCeremonyColumnVisibility] =
+		React.useState<VisibilityState>({});
+	const [ceremonyRowSelection, setCeremonyRowSelection] = React.useState({});
+	const [ceremonyPagination, setCeremonyPagination] = React.useState<PaginationState>({
+		pageIndex: 0,
+		pageSize: 5,
+	})
+
+	const passportTable = useReactTable({
+		data: passportData,
+		columns: passportColumns,
+		onSortingChange: setPassportSorting,
+		onColumnFiltersChange: setPassportColumnFilters,
+		getCoreRowModel: getCoreRowModel(),
+		getPaginationRowModel: getPaginationRowModel(),
+		getSortedRowModel: getSortedRowModel(),
+		getFilteredRowModel: getFilteredRowModel(),
+		onColumnVisibilityChange: setPassportColumnVisibility,
+		onRowSelectionChange: setPassportRowSelection,
+		onPaginationChange: setPassportPagination,
+		state: {
+			sorting: passportSorting,
+			columnFilters: passportColumnFilters,
+			columnVisibility: passportColumnVisibility,
+			rowSelection: passportRowSelection,
+			pagination: passportPagination,
+		},
+	});
+
+	const ceremonyTable = useReactTable({
+		data: ceremonyData,
+		columns: ceremonyColumns,
+		onSortingChange: setCeremonySorting,
+		onColumnFiltersChange: setCeremonyColumnFilters,
+		getCoreRowModel: getCoreRowModel(),
+		getPaginationRowModel: getPaginationRowModel(),
+		getSortedRowModel: getSortedRowModel(),
+		getFilteredRowModel: getFilteredRowModel(),
+		onColumnVisibilityChange: setCeremonyColumnVisibility,
+		onRowSelectionChange: setCeremonyRowSelection,
+		onPaginationChange: setCeremonyPagination,
+		state: {
+			sorting: ceremonySorting,
+			columnFilters: ceremonyColumnFilters,
+			columnVisibility: ceremonyColumnVisibility,
+			rowSelection: ceremonyRowSelection,
+			pagination: ceremonyPagination,
+		},
+	});
+
+	const [ceremonyTime, setCeremonyTime] = useState("noPassportCeremony");
+	const [modifyCeremonyTime, setModifyCeremonyTime] =
+		useState("noPassportCeremony");
+	const [deleteCeremonyTime, setDeleteCeremonyTime] =
+		useState("noPassportCeremony");
+
+	return (
+		<div className="w-full">
+			<p>
+				&quot;But wait!&quot; I hear you saying. &quot;Isn&apos;t everything
+				just in random places?&quot;
+			</p>
+			<p>Yes. :D</p>
+			<br />
```

Comment:
should prob be removed lol

Reply:
Forgot about this lmao

Reply:
Resolved in [00d29d9](https://github.com/purduehackers/passport-issuing-office/pull/52/commits/00d29d972990892c27894b10c9ce4850b3c19ed8)